### PR TITLE
Add: philosophy style

### PR DESCRIPTION
**Pull request by toorusr**
> It does not seem to be possible to style the markdown using tailwindcss, so I have added a docs css in which all styles have been declared specifically.  

## What I did
1. Created a `docs.css` and linked it in the `masters.blade.php` layout as a stylesheet.
2. Implemented docs class in the content div in `philosophy.blade.php`.
3. Styled the `philosophy.md` as good as I can using pure css.

## What needs to be done
The style really does need improvement, because I am not the best designer and I bet there are better ones out there.

### CODE REVIEW

File: source/_layouts/philosophy.blade.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 @section('body')
     <main class="p-8 bg-blue">
         <div class="text-3xl mb-4 text-white">Philosophie der Camps</div>
-            <div class="p-8 rounded bg-white">
+            <div class="p-8 rounded bg-white docs">
                 @yield('philosophy_content')
```
Comment (minthemiddle) [inline source/_layouts/philosophy.blade.php]: That's where the content can go to to enable styling…

**Comment by minthemiddle** [comment]
Oh I see what the problem is. There's no need to use Markdown here. Just use `philosophy.blade.php` for both content and class styling. Just put it in where `@yield` is. There's no win using Markdown when we want to massively style the output…

**Comment by toorusr** [comment]
Why is there no need to use markdown here? It is much easier to edit and you can quickly style your document. There is definitely a win.

**Comment by minthemiddle** [comment]
Well, the idea of a utility based CSS system like Tailwind is to not style things via CSS directly but with classes. Once we have a skilled designer who can make the site shine, she only needs to update Tailwind's config once and the design appears everywhere on the page – except when you introduce own CSS as in this PR.

That is the huge disadvantage of Markdown here: You cannot use Tailwind's utility classes. With the custom CSS you introduce, whenever we make global design changes, we need to remember that we customized `philosophy` (and will mostly not do so).

Also, for long form text, markdown can be very tricky. You have to add `  ` (two spaces) to paragraphs to get a custom line break. Empty lines need two spaces as well so you get a proper `<p>` paragraph. Of course you can also just sprinkle in HTML tags in Markdown. But that makes a huge mess.

Consider the alternative:

Instead of `# Heading` you have `<div class="text-xl">Heading</div>`, and for paragraphs you have some `<p>`s. The markup is still light, but we can assign utility classes and have one less thing to worry about once we get a proper design…

**Comment by toorusr** [comment]
Good Idea

**Comment by Jubeki** [comment]
What happened to this pull request?
Can it be closed or is it still needed?

**Comment by minthemiddle** [comment]
Outdated PR. Philosophy style should be updated via Tailwind classes.

**Pull request closed without merging**